src/js/components/tasks/table.tsx

```diff
@@ -352,15 +407,15 @@ const TaskTable = ({
           </>
         }
         property="status"
-        width="20%"
+        width={viewEpicsColumn ? '20%' : '30%'}
```

Comment:
@stacyk When `viewEpicsColumn` is true, these widths add up to 100%. When it's false though, they add up to 130% -- is that a problem?

Reply:
@jgerigmeyer I wonder if there is a condition/place I am not testing because I get 100% on the epic table (which is the other table in the tab so not involved in this file), 100% on the task table in the tab, then when on the Epic page, the task table is 100% (that is where we don't have an epic column). 

Also, should those data-label attributes say `data-label=[object, Object]`?

![task-table-in-tab](https://user-images.githubusercontent.com/1581694/126050635-53348227-a713-4a0b-a0b8-4a263183facd.jpg)
![task-table-on-epic-page](https://user-images.githubusercontent.com/1581694/126050636-01491649-db58-414a-a7ca-cc3d3899c1fc.jpg)
![epic-table-tab](https://user-images.githubusercontent.com/1581694/126050637-8ac4a698-905a-49d5-8606-f119dcc5391e.jpg)

Reply:
@stacyk Ah, my mistake -- you're totally right. It's 100% throughout.

> Also, should those data-label attributes say `data-label=[object, Object]`?

Yeah, I've seen that too. Unfortunately that's added by React SLDS and we have no control over it -- I tried overriding it unsuccessfully.